homework/test/test-list.js

```diff
@@ -4,5 +4,6 @@ describe('Test > ', function() {
 
   it('will test some pure functions', function() {
     // Write any pure function assertion here.
+      assert.deepEqual(Math.sin(0) , 0);
```

Comment:
It's OKAY. Although it could be better if you define a method in one manager with such content, and then call it here.